codecommit-put-file-action/README.md

```diff
@@ -0,0 +1,61 @@
+# Action to put a file in a Codecommit repo 🔄
+
+This action uses the [vanilla AWS CLI](https://docs.aws.amazon.com/cli/index.html) to put file in the codecommit repo.
+
+
+## Usage
+
+### `workflow.yml` Example
+
+Place in a `.yml` file such as this one in your `.github/workflows` folder. [Refer to the documentation on workflow YAML syntax here.](https://help.github.com/en/articles/workflow-syntax-for-github-actions)
+
+```
+name: Codecommit put-file
+on: push
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@master
+    - name: Put file into a codecommit repo
+      uses: aws-robotics/robomaker-sample-app-ci/codecommit-put-file-action@v2.0.0
+      env:
+        AWS_REGION: 'us-east-2'
```

Comment:
Does region matter for CodeCommit?

Reply:
Yep AWS_REGION is not required.. made the changes accordingly